src/manifestoo/main.py

```diff
@@ -225,10 +226,23 @@ def list(
         None,
         help="Separator character to use (by default, print one item per line).",
     ),
+    sorting: str = typer.Option(
+        "alphabetical",
+        "--sorting",
+        "--sort",
+        help=(
+            "Choice between 'alphabetical' and 'topological'."
```

Comment:
```suggestion
            "Choice between 'alphabetical' and 'topological'. "
```

Also for the 2 others